### PR TITLE
Expiration -> Timestamp

### PR DESCRIPTION
**Pull request by shanev**

### CODE REVIEW

**Review by yubrew — APPROVED** [review]
👍

File: contracts/whitelist/src/contract.rs [hunk 1]
```diff
@@ -105,24 +108,21 @@ pub fn instantiate(
...
-    let genesis_start_time = Expiration::AtTime(Timestamp::from_nanos(GENESIS_MINT_START_TIME));
+    let genesis_start_time = Timestamp::from_nanos(GENESIS_MINT_START_TIME);
     if msg.start_time < genesis_start_time {
         return Err(ContractError::InvalidStartTime(
-            msg.start_time,
-            genesis_start_time,
+            start_time,
+            Expiration::AtTime(genesis_start_time),
         ));
     }
```
Comment (jhernandezb) [inline contracts/whitelist/src/contract.rs:127]: a bit confused of the mix in using expiration and timestamps, maybe adding comments on which one is being used for new readers of the code.

File: contracts/minter/src/contract.rs [hunk 2]
```diff
@@ -432,14 +430,14 @@ pub fn execute_update_start_time(
         return Err(ContractError::AlreadyStarted {});
     }
 
-    let genesis_start_time = Expiration::AtTime(Timestamp::from_nanos(GENESIS_MINT_START_TIME));
+    let genesis_start_time = Timestamp::from_nanos(GENESIS_MINT_START_TIME);
     let start_time = if start_time < genesis_start_time {
         genesis_start_time
     } else {
         start_time
```
Comment (jhernandezb) [inline contracts/minter/src/contract.rs:437]: we are not checking that the new `start_time > env.block` time right ?

**Review by jhernandezb — CHANGES_REQUESTED** [review]

File: contracts/minter/src/contract.rs (hunk 2 above)
Comment (jhernandezb) [reply contracts/minter/src/contract.rs:437]: also maybe for updating doesn't makes sense to default to genesis time, just in instantiation. You have to be explicit about the time you want to update to.

Comment (shanev) [reply contracts/minter/src/contract.rs]: Also fixed this.

**Review by jhernandezb — APPROVED** [review]

**Pull request merged**